next.config.js

```diff
@@ -9,6 +9,15 @@ const config = {
   distDir: process.env.DIST_DIR || 'dist',
   poweredByHeader: false,
   reactStrictMode: true,
+  async rewrites() {
```

Comment:
I am getting this error with `yarn build`
`Error: rewrites must return an array, received undefined.`